.github/workflows/triage.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  issues:
+    types:
+    - opened
+    - reopened
+  pull_request:
+    types:
+    - opened
+    - reopened
+jobs:
+  issues:
+    if: github.event_name == 'issues'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: andymckay/labeler@1.0.2
```

Comment:
Looks good 👍🏼  Are these actions/images safe and widely used?

Reply:
Yeah, I found them in the GHA docs at: 
* [adding labels to issues](https://docs.github.com/en/actions/managing-issues-and-pull-requests/adding-labels-to-issues)
* [moving assigned issues to project](https://docs.github.com/en/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards)

The labeler action is used by more than 6.3k pipelines, per their GH repo stats.

Reply:
same question

Reply:
These are the ones used in the GHA docs. Most of the other actions found in the marketplace are not endorsed by GitHub. There are some more "official looking" ones, with tests:

* https://github.com/actions-ecosystem/action-add-labels
* https://github.com/actions-ecosystem/action-create-comment
* https://github.com/actions/add-to-project (I think this works only with Projects Beta)

Other than that, they are just as old, with similar number of issues, forks, stars etc.

Reply:
Do we have any additional selection criteria?

Reply:
I changed the 'labelling' and 'commenting' action plugins to the ones from `actions-ecosystem`. If anything, they at least have tests to cover their code.